src/api.js

```diff
@@ -1,199 +1,102 @@
-'use strict';
```

Comment:
I might have just deleted it on accident. That said, I'd like to have babel do this for us so we don't have to remember.

https://babeljs.io/docs/plugins/transform-strict-mode/